roles/default/kiali-deploy/templates/openshift/role.yaml

```diff
@@ -86,6 +89,7 @@ rules:
   - get
   - list
   - watch
+  - patch
```

Comment:
Need to add "patch" here to the operator role, line 259: https://github.com/kiali/kiali-operator/blob/v1.21/deploy/role.yaml#L259